ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php

```diff
@@ -138,7 +141,7 @@ public function completeCatchEvent(Definitions $definitions, ExecutionInstanceIn
         $this->validateData($data, $definitions, $element);
 
         // Get complementary information
-        $version = $definitions->getLatestVersion();
+        $version = $instance->process_version_id;
```

Comment:
you need to change the line: `'bpmn' => $version->getKey(),` to `'bpmn' => $version,`

Reply:
Nice catch, fixed.